images/image.mk

```diff
@@ -17,7 +17,9 @@
 
 override GOOS=linux
 
+ifeq ($(origin DOCKERCMD),undefined)
```

Comment:
Is this to enable podman? How about a separate commit or PR?

Reply:
:) yes, that one change was for podman. I will separate the commit out.

Reply:
Fixed.